db/deploy/tables/applications.sql

```diff
@@ -5,10 +5,63 @@ begin;
 
 create table if not exists connectivity_intake_public.applications (
   id serial not null,
+  owner uuid not null,
```

Comment:
We don't actually want this to be a mandatory field in our mutation, so we actually don't want to enforce here the column is not null. Let's make a ticket to explore future options to improve this code.

```suggestion
  owner uuid,
```